azure-steps-windows14.yml

```diff
@@ -0,0 +1,76 @@
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: $(PYTHON_VERSION)
+    addToPath: true
+    architecture: $(PYTHON_ARCH)
+
+- script: python -m pip install --upgrade pip wheel
+  displayName: 'Install tools'
+
+- script: python -m pip install -r test_requirements.txt
+  displayName: 'Install dependencies; some are optional to avoid test skips'
+
+- powershell: |
+    $ErrorActionPreference = "Stop"
+    # Download and get the path to "openblas.a". We cannot copy it
+    # to $PYTHON_EXE's directory since that is on a different drive which
+    # mingw does not like. Instead copy it to a directory and set OPENBLAS,
+    # since OPENBLAS will be picked up by the openblas discovery
+    $target = $(python tools/openblas_support.py)
+    mkdir openblas
+    echo "Copying $target to openblas/"
+    cp $target openblas/
+  displayName: 'Download / Install OpenBLAS'
+
+- task: BatchScript@1
+  inputs:
+    filename: tools/set_vs_toolchain.cmd
+    arguments: $(BITS) 2019 14.16
+    modifyEnvironment: True
+  displayName: Set VS toolchain version
+
+- powershell: |
+    ls openblas
+    echo What PATH and other environment variables?
+    echo PATH $Env:PATH
```

Comment:
For those trying to follow what happened, the PATH printed here in CI is not being modified by the previous BatchScript step, so the older 141 toolset is not used. You can see that the PATH printed at the end of `tools/set_vs_toolchain.cmd` is not the same as the one printed here.